versioned_docs/version-2.0/how_to_guides/datasets/manage_datasets_in_application.mdx

```diff
@@ -72,3 +72,22 @@ To do so, select a dataset, click on "Examples", and then click the "Export Data
 This will open a modal where you can select the format you want to export to.
 
 ![Export Dataset Modal](../static/export-dataset-modal.png)
+
+## Create and manage dataset splits
+
+Dataset splits are divisions of your dataset which you can use to segment your data. For example, it is common
+in machine learning workflows to split datasets into test, training, and validation sets. This can be useful
+to prevent overfitting - where a model performs well on the training data but poorly on unseen data. In evaluation
+workflows, it can be useful to do this when you have a dataset with multiple categories that you may want to evaluate
+separately; or if you are testing a new use case that you may want to include in your dataset in the future, but want
+to keep separate for now.
+
+In machine learning, it is best practice to keep your splits separate (each example belongs to exactly one split).
+However, we allow you to select multiple splits for the same example in LangSmith because it can make sense for
+some evaluation workflows - for example, if an example falls into multiple categories on which you may want to
```

Comment:
that feels weird to me... have never seen it done before. i get that fuzzy membership is a thing, just feels not useful and could potentially lead to a mistake if we don't render it nicely

Reply:
hopefully the rendering is good enough to solve this? They will show up in the examples table like this (first one is in multiple splits):
![Screenshot 2024-05-20 at 3 56 00 PM](https://github.com/langchain-ai/langsmith-docs/assets/6432132/649b7b84-7c47-46ae-8da8-951f148505da)

Reply:
do we have a default split?